app/src/main/res/drawable/materialdrawer_shape_small.xml

```diff
@@ -0,0 +1,5 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle" >
+    <solid android:color="#000"/>
+    <corners android:radius="5dp"/>
```

Comment:
unfortunately its not possible to use % here, so i had to create two files, one for the selected profile, one for the others